### DISCUSSION TITLE
Is it possible to add a second hub network within the same region?

### DISCUSSION THREAD

**Discussion by benoitmonnier** [Azure/Enterprise-Scale#1130] [Q&A] [answered]
Hello everyone. Is it possible to add a second hub network within the same region ?

Exemple : 

- A hub network for a production environment in central France
- A hub network for a testing environment in central France

Thank you in advance,

Regards

**Comment by jtracey93** [comment, marked as answer]
Hey @benoitmonnier,

Yes it sure is supported and possible. This is just not part of the accelerators

**Comment by benoitmonnier** [comment]
Thank you @jtracey93 . So I have no choice to write my own terraform code to do this, right ?

**Reply by jtracey93** [reply]
If you are using the `caf-enterprise-scale` Terraform module, it may be worth you opening an issue other there to discuss this further around specific implementation details. https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues

Looks like it may be very similar to https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues/328 and looks like is a workaround in there also 👍

**Comment by benoitmonnier** [comment]
This is exactly what I needed. Thank you a lot :)
